### ISSUE TITLE
Splash Screen stuck in wallet startup process

### ISSUE THREAD

**Issue by SativaL** [particl/partgui#330]
The Splash Screen with the three dots get stucked upon startup (the wallet startup to enter overview takes around 1-2 seconds normally, and sadly the 3 dots are aways stuck and dont move (looks bugged).

Would be awesome to have them working still even if its only 2 second startup time. 

if needed i can provide video/screen

**Comment by LiteBit** [comment]
Please attach a video or gif showing what you're seeing. And the OS you're on. Thanks

**Comment by pciavald** [comment]
no need for a gif, it's doing it on all versions. It seems this part of the bootstraping is not multi-threaded and all the resource is taken by the GUI starting, avoiding the GIF to run...
is that correct @kewde ?

**Comment by rynomster** [comment]
Yeah unfortunately it's not possible to fix this. The main thread is blocked by angular at this point in time.

**Comment by pciavald** [comment]
Actually, knowing that it is a wontfix, can we remove the gif ? @allienworks @gerlofvanek

**Comment by SativaL** [comment]
what does this mean for future builds ? (idk maybe the whole syncing process

**Comment by allienworks** [comment]
Animation removed, PR waiting to be merged.
